.github/actions/download-rootfs/download-rootfs.ps1

```diff
@@ -63,9 +63,14 @@ function Test-Checksums {
 
     # Parse checksum file
     $image = $URL.Segments[$Url.Segments.Length - 1]
-    $pattern = "(\w+)  ${image}"
+    $pattern = "(\w+) \*${image}"
+    $matched = (Select-String -Pattern "${pattern}" -Path "${Checksums}").Matches
+    if ($null -eq $matched -or $matched.Count -eq 0) {
+        Write-Warning "Could not find $image in checksums file"
+        return $false
+    }
 
-    $newSHA256 = (Select-String -Pattern "${pattern}" -Path "${Checksums}").Matches[0].Groups[1]
+    $newSHA256 = $matched.Groups[1]
```

Comment:
I'm not very familiar with powershell, is there a reason `$matched` effectively went from `...Matches[0]` to just `...Matches`?

Reply:
Me messing with the working code during rebases...